extensions/spring-xd-extension-sqoop/src/main/java/org/springframework/xd/sqoop/SqoopRunner.java

```diff
@@ -50,7 +50,7 @@ public static void main(String[] args) {
 			throw new IllegalArgumentException("Missing arguments and/or configuration options for Sqoop");
 		}
 
-		String jarPath = com.cloudera.sqoop.util.Jars.getJarPathForClass(JobConf.class);
+		String jarPath = org.apache.sqoop.util.Jars.getJarPathForClass(JobConf.class);
```

Comment:
This looks a bit odd, org.apache sounds better than using a cloudera artifact, but who knows, there might have been a real reason for it.    @trisberg @jvalkeal what do you think